src/epplibwrapper/client.py

```diff
@@ -163,7 +163,8 @@ def send(self, command, *, cleaned=False):
             try:
                 return self._send(command)
             except RegistryError as err:
-                if err.should_retry() and counter < 3:
+                if counter < 3 and (err.should_retry() or err.is_transport_error()):
+                    logger.info(f"Retrying transport error. Attempt #{counter+1} of 3.")
```

Comment:
```suggestion
                    logger.info(f"Retrying command, an exception was encountered. Attempt #{counter+1} of 3.")
```
Since we can get errors other than just TransportError, maybe something like above would be more fitting? Ideally it should be for should_retry and is_transport_error, but I think something generic is OK too as long as we have a way of back tracking what error was thrown